assets/service_now_notifications_handler/index.py

```diff
@@ -513,14 +598,39 @@ def _extract_event_body(self, event):
         """
         try:
             # Extract the request body from API Gateway event
-            body = event.get('body', '{}')
-            if event.get('isBase64Encoded', False):
+            body = event.get("body", "{}")
+
+            # Log the raw body for debugging
+            logger.debug(f"Raw body: {body}")
```

Comment:
question(blocking): How likely is it that there is sensitive information in the body of the event? If there is any likelihood, how are we handling that situation?